### ISSUE TITLE
Create example input and output files to include in our end user facing documentation

### ISSUE THREAD

**Issue by davidatbu** [davidatbu/openFraming#171]
As @monajalal  pointed out, it would be great to have example input and output files as part of our documentation.

### Classifier related
1. Input for `POST /classifiers/training/file` (ie, a training set).
2. Input for  `POST /classifiers/<int:classifier_id>/test_sets/file` . (ie, a test set)
3. Output of `GET /classifiers/<int:classifier_id>/test_sets/predictions` . (ie,  test set prediction results)

### Topic model related
1. Input to  `POST /topic_models/<int:topic_model_id>/training/file` (a training set).
2. Output of `GET /topic_models/<int:topic_model_id>/keywords` (important keywords for each topic).
3.  Output of `GET /topic_models/<int:topic_model_id>/topics_by_doc` (topic proportions for each example).

These files should be consistent. That is, The output files must be produced by the system after feeding the system with the input files.

**Comment by davidatbu** [comment]
Here's one for topic models. [50,000 headlines for topic model training](https://termbin.com/bls1)
